Sources/SWBCore/Extensions/DeveloperDirectoryExtension.swift

```diff
@@ -0,0 +1,25 @@
+//===----------------------------------------------------------------------===//
```

Comment:
This needs to be added to the make build

Reply:
Thanks for catching this!